projects/openh264/decoder_fuzzer.cpp

```diff
@@ -0,0 +1,90 @@
+// TODO: This should be moved to the openh264 repo.
+
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "codec_def.h"
+#include "codec_app_def.h"
+#include "codec_api.h"
+#include "read_config.h"
+#include "typedefs.h"
+#include "measure_time.h"
+
+/*
+ * To build locally:
+ * CC=clang CXX=clang++ CFLAGS="-fsanitize=address,fuzzer-no-link -g" CXXFLAGS="-fsanitize=address,fuzzer-no-link -g" LDFLAGS="-fsanitize=address,fuzzer-no-link" make -j$(nproc) USE_ASM=No BUILDTYPE=Debug libraries
+ * clang++ -o decoder_fuzzer -fsanitize=address -g -O1 -I./codec/api/svc -I./codec/console/common/inc -I./codec/common/inc -L. -lFuzzer -lstdc++ decoder_fuzzer.cpp libopenh264.a
+ */
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+  int32_t i;
+  int32_t iBufPos = 0;
+  int32_t iEndOfStreamFlag;
+  int32_t iSliceSize;
+  ISVCDecoder *pDecoder;
+  SDecodingParam sDecParam = {0};
+  SBufferInfo sDstBufInfo;
+  uint8_t* pBuf;
+  uint8_t* pData[3] = {NULL};
+  uint8_t uiStartCode[4] = {0, 0, 0, 1};
+
+  pBuf = new uint8_t[size + 4];
```

Comment:
You can save yourself a bunch of manual free logic by making this a `std::unique_ptr<uint8_t[]>`